File: Library/Database/Connection.php
<?php

class Connection
{
    private $conn;

    /**
     * Database constructor.
     *
     *
     * @param Config $config
     *
     * @throws ConnectionException
     */
    public function __construct(Config $config)
    {
        $conn = new mysqli($config->host, $config->user, $config->pass, $config->dbname);

        if ($conn->connect_error) {
            throw new ConnectionException($conn->connect_error);
        }
    }


}

File: lib/database/Connection.py
import pymysql

from lib.database.Config import Config


class Connection:
    def __init__(self, config: Config):
        self.conn = pymysql.connect(
            host=config.host,
            user=config.user,
            password=config.password,
            database=config.database
        )

    def getCursor(self):
        return self.conn.cursor(pymysql.cursors.DictCursor)
